Views/Person/Create.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@model Personaltool.Models.Person
+
+@{
+    ViewData["Title"] = "Create";
+}
+
+<h1>Create</h1>
+
+@{await Html.RenderPartialAsync("_CreatePartial", Model);}
```

Comment:
I think it's better to have the partial centered instead of pushing it all the way left